src/planner/operator/persistent/logical_copy_from.cpp

```diff
@@ -6,15 +6,29 @@ namespace kuzu {
 namespace planner {
 
 void LogicalCopyFrom::computeFactorizedSchema() {
-    copyChildSchema(0);
+    if (getNumChildren() == 1) {
```

Comment:
hmm do we really have to check num children to identify if this is copy rdf? If so, better to keep a field inside LogicalCopyFrom to indicate of copy type (NODE, REL, RDF).